### DISCUSSION TITLE
Transfer ownership of extension in Partner Center from one developer account to another

### DISCUSSION THREAD

**Discussion by varaduMSFT** [microsoft/MicrosoftEdge-Extensions#15] [Q&A] [answered]
Currently there is no way to transfer ownership of Microsoft Edge Addons within Microoft Partner Center. 

An extension developer may want to transfer ownership of the Edge extension after it has been built, for various reasons such as
 - the developer built the extension to cater to a requirement of his  or her customer, and the extension should belong to that customer
 - the developer has to hand over ownership of an enterprise extension when he or she is changing teams within their organization

**Comment by drveresh** [comment]
I am one of the very first person who raised this request and waiting for it since 2019.  I have had several discussions justifying its purpose through numerous emails and meeting calls with your team, yet noting is materialized.

The feature of transferring extensions or add-ons between two user accounts has been there since 2009 on several platforms like Chrome, Firefox, Apple, and more, and yet it is still missing from Edge for too long.

Please pause on to the cosmetic and fancy features, instead work on this fundamental feature ASAP!

Please explain to the community why you are not prioritizing them when every other platform store is offering this for years since day 1?

**Comment by drveresh** [comment]
@varaduMSFT Can you please share the latest updates on this issue?

**Comment by drveresh** [comment]
@varaduMSFT I have been following up on this fundamental issue since 2019 and have already sent numerous emails explaining the criticality of this feature, and yet nothing is happening.  Please **share the timelines** if there is any real and concrete plan for this.

**Comment by drveresh** [comment]
@varaduMSFT @varaduMSFT @nagachaitanyalokam
Can one of you respond to this very prolonged issue?

**Reply by drveresh** [reply]
If you cannot respond to discussions, then at least take an action via this issue - [Extensions/issues/74](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/74)

**Comment by Deliaz** [comment]
Are there any updates regarding the ownership transfer?

**Comment by drveresh** [comment]
@varaduMSFT @sivsouvamMSFT Can you please respond and confirm? As you promised last year (actually since 2019), it should be been ready by now.  You cannot delay this forever.

**Reply by Bhuvana-Priya** [reply]
Hi @drveresh ,

We apologize for the delay, our teams have this in pipeline. However, due to other priorities, this has gotten delayed. We will let you know as soon as there is an update.

**Reply by drveresh** [reply]
@Bhuvana-Priya What is the latest update on this? I still don't see any official update on this prolonged (since 2019) issue.

**Reply by Bhuvana-Priya** [reply]
Hi @drveresh, This is definitely in our roadmap and we are testing it out. We will let you know once this is made official.

**Reply by drveresh** [reply]
@Bhuvana-Priya Your team has been making such futile statements since 2019, and not showing any sense of seriousness and prolonging this year after year. In the last five years, hundreds of super-advanced technologies have been launched, like ChatGPT, GeminiAI, SpaceShip, BingAI, etc., but your team failed to release this one basic feature despite cloning the Chromium project.  Who drafted this poor roadmap that is failing to deliver a single feature?  What is there to test this basic feature?  What is the concrete definition(timelines) of "once" here?

**Comment by sandeepchads** [comment]
Folks, we are supporting ownership transfer in the backend. We request you to kindly submit your request to Microsoft Edge Extensions Developer Support <ext_dev_support@microsoft.com> if you have a requirement to transfer ownership.

**Comment by ManikanthMSFT** [comment]
Hi @drveresh, could you please confirm if the requirement mentioned in the query is still needed as we now have a work around for this query?

**Reply by drveresh** [reply]
@ManikanthMSFT Thanks for the follow-up and it is no longer required.  Thanks a lot for your support.  However, when will this feature be released publicly?

**Reply by ManikanthMSFT** [reply]
@drveresh, if any developer wishes to transfer their ownership, they can email us, and we will proceed with the necessary steps.

**Reply by Stitch86** [reply]
if we try to transfer one of our extension ownership, which email address we can send to?

**Reply by ManikanthMSFT** [reply]
Hi @Stitch86, we request you to kindly submit your request to [Microsoft Edge Extensions Developer Support](ext_dev_support@microsoft.com) if you have a requirement to transfer ownership.

**Comment by Rahul-Bauri** [comment, marked as answer]
As an extension developer, you can now seamlessly transfer ownership of your Microsoft Edge extension to another party. Transferring ownership of an extension can be essential for various reasons, such as organizational changes, project handovers, or streamlining management processes. Your Microsoft Edge extension can continue to thrive under new ownership without any disruptions.
 
Learn more about how to **Transfer ownership of an extension:**
 
https://learn.microsoft.com/en-us/microsoft-edge/extensions/update/transfer-ownership
